api/external/events.html

```diff
@@ -0,0 +1,530 @@
+                            </dt>
+                            <dd>
+                                <p>Create a member event type.</p>
+                                <table class="docutils field-list" frame="void" rules="none">
+                                    <col class="field-name" />
+                                    <col class="field-body" />
+                                    <tbody valign="top">
+                                        <tr class="field-odd field"><th class="field-name">Parameters:</th>
+                                            <td class="field-body">
+                                                <ul class="first simple">
+                                                    <li><strong>event_type</strong> (<em>string</em>) &#8211; Required. The type of the event.</li>
+                                                    <li><strong>display_name</strong> (<em>string</em>) &#8211; Required. A name for displaying the event in the UI.</li>
+                                                </ul>
+                                            </td>
+                                        </tr>
+                                        <tr class="field-even field">
+                                            <th class="field-name">Returns:</th>
+                                            <td class="field-body">
+                                                <p class="first">Information for the new member event type.</p>
+                                            </td>
+                                        </tr>
+                                        <tr class="field-even field">
+                                            <th class="field-name">Raises:</th>
+                                            <td class="field-body">
+                                                <tt class="docutils literal">
+                                                    <span class="pre">Http400</span>
+                                                </tt>
+                                                if no display name or event type are provided.
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                                <p>
+                                    <div class="sample-response hide-response">
+                                        <span style="font-weight:bold;">Sample Response</span>
+                                        <a href="#" class="resp-trigger">[<span class="tshow">show</span><span class="thide">hide</span>]</a>
+<pre class="response">
+    <b>POST /100/events</b>
+    {
+        "account_id": 100,
+        "created_at": "@D:2025-11-24T10:23:10",
+        "display_name": "Visited Website",
+        "event_type": "visited_website",
+        "event_type_id": 300
```

Comment:
- `account_id` is interpreted from the url
- `created_at` is set on DB write
- `event_type_id` is generated by the db

Since these values are not taken from the request body, we should remove them from the sample request body


```suggestion
        "display_name": "Visited Website",
        "event_type": "visited_website",
```

Reply:
@Streeter8 This is the example response body, not request body.

Theese are the params described for this endpoint.
```
<td class="field-body">
  <ul class="first simple">
      <li><strong>event_type</strong> (<em>string</em>) &#8211; Required. The type of the event.</li>
      <li><strong>display_name</strong> (<em>string</em>) &#8211; Required. A name for displaying the event in the UI.</li>
  </ul>
</td>
```

And then this is the sample response:

```
{
    "account_id": 100,
    "created_at": "@D:2025-11-24T10:23:10",
    "display_name": "Visited Website",
    "event_type": "visited_website",
    "event_type_id": 300
}
```